README.md

```diff
@@ -117,5 +121,17 @@ If the Hammer Test is performed during another player's turn and that player giv
 If a Lord drops their hammer during the Flip phase they are instantly demoted to Lad. Before ending their turn they must drive a new starting nail into the outer ring. 
 
 
+### Uprising
+
+At the start of each game, there is an Uprising counter that begins as 0. A di or some other object may be used to represent this or it can be kept track of mentally by all players as a group.
+
+If the active player is a Lad, they may call "Uprising" before flipping their hammer. If they do then the turn officially counts as an Uprising. The player must perform a double hammer flip followed by one strike without hesitation (note that they do not get two strikes like they normally would). If they hit their nail with that strike then the Uprising was successful and the Uprising counter is incremented by 1. If they drop the hammer, regrip the hammer, or miss their nail the Uprising counter is reset to 0 in which case they must drink and end their turn immediately, passing the hammer to the next player in turn order.
```

Comment:
`If they hit their nail with that strike, then the Uprising was successful, and the Uprising counter is incremented by 1.`